static/sass/6-components/_footer.scss

```diff
@@ -1,6 +1,14 @@
 .c-site-footer {
-  .border--yellow_notch {
-    border-top: 2px solid $color-yellow-tribune;
-    margin: $size-xs 0;
+  
+  &__logo {
+    font-size: 4rem;
+    margin-bottom: 2.6rem;
```

Comment:
Could use a utility class here, I think. I use `has-b-btm-marg` in the portal.

Reply:
good catch!